DICOM/src/Microsoft.Health.DeID.SharedLib/EncryptFunction.cs

```diff
@@ -69,7 +82,7 @@ public static byte[] EncryptContentWithAES(byte[] plainBytes, byte[] key)
             return result;
         }
 
-        public static byte[] DecryptContentWithAES(string cipherText, byte[] key)
+        public byte[] DecryptContentWithAES(string cipherText)
```

Comment:
>string cipherText [](http://example.com/codeflow?start=44&length=17)

It's a little weird here.
How could a string represent a cipher data?
I assume not all binary data could be decoded as a valid UTF-8 string.
What's the use case of this function?